### PR TITLE
[DX-1520] Add explanation to the use of catalogue and catalog

### PR DESCRIPTION
**Pull request by letzya**
https://deploy-preview-5084--tyk-docs.netlify.app/docs/nightly/tyk-apis/tyk-portal-api/portal-documentation

https://deploy-preview-5084--tyk-docs.netlify.app/docs/nightly/tyk-portal-api

### CODE REVIEW

File: tyk-docs/content/tyk-apis/tyk-portal-api/portal-documentation.md [hunk 1]
```diff
@@ -1,12 +1,27 @@
 ---
 date: 2017-03-27T12:28:24+01:00
-title: Portal Documentation
-menu:
-  main:
-    parent: "Tyk Portal API"
-weight: 6
+title: API Publishing Endpoints
+description: "Descript the endpoint API admin or owner can use to publishing their API in the developer portal"
```
Comment (yurisasuke) [inline tyk-docs/content/tyk-apis/tyk-portal-api/portal-documentation.md]: should this be describe ?

File: tyk-docs/content/tyk-portal-api.md [hunk 2]
```diff
@@ -1,13 +1,10 @@
...
-  main:
-    parent: "Tyk Portal Classic"
-weight: 8
+title: Classic Portal API
+description: "Landing page for the Tyk Classic Portal API documentation"
+tags: ["Tyk Classic Portal API", "Classic Portal API "]
 ---
 
-The Tyk Portal API covers all available endpoints for your developer portal, including:
+This section describe the Tyk Classic Portal API endpoints for your classic developer portal. It includes the following:
```
Comment (yurisasuke) [inline tyk-docs/content/tyk-portal-api.md]: I think this sentence should be. I dont thik the  `for your classic developer portal` is necessary
This section describe the Tyk Classic Portal API endpoints. It includes the following:

What do you think?

**Review by yurisasuke — APPROVED** [review]

File: tyk-docs/content/tyk-apis/tyk-portal-api/portal-documentation.md (hunk 1 above)
Comment (letzya) [inline tyk-docs/content/tyk-apis/tyk-portal-api/portal-documentation.md]:
```suggestion
description: "This page details the API endpoint used for publishing APIs to Tyk classic Dev Portal. API platform teams and API owners can use this endpoint to integrate their APIs, making them visible and accessible to developers."```

File: tyk-docs/content/tyk-portal-api.md (hunk 2 above)
Comment (letzya) [inline tyk-docs/content/tyk-portal-api.md]:
```suggestion
This section describes the Tyk Classic Portal API endpoints. It includes the following:
```

**Pull request merged**